### PR TITLE
[MODUIMP-38] fix serialization of mod-users responses

### PR DESCRIPTION
**Pull request by wwelling**
[MODUIMP-38](https://issues.folio.org/projects/MODUIMP/issues/MODUIMP-38?filter=allopenissues)

### CODE REVIEW

**Comment by doytch** [comment]
Is there a related ticket or write-up about this somewhere? I'll be bashing on mod-user-import again when we get Honeysuckle at Duke and am trying to get a feel for what I'm in for with this. 😆

**Comment by wwelling** [comment]
@doytch there is now

**Review by dcrossleyau — APPROVED** [review]
I add example departments, and verified the RAML/Schema.

Thanks for adding a property description.

**Comment by dcrossleyau** [comment]
I have contacted DevOps Slack channel about the strange failure of this PR.
The individual stages are successful, but overall it now says that there are failed tests.

**Comment by wwelling** [comment]
@dcrossleyau thanks for adding the example departments. I should have remembered to update the sample.

**Comment by dcrossleyau** [comment]
I temporarily disabled the repo "Settings : Branch protection". So when you are ready, then this can be merged despite the "failed" test for "pr-merge".

File: ramls/schemas/userdataimport.json [hunk 1]
```diff
@@ -34,6 +34,14 @@
...
       }
     },
+    "departments": {
+      "description": "Names of departments the user belongs to",
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "type": "string"
+      }
+    },
```
Comment (julianladisch) [inline ramls/schemas/userdataimport.json:44]: This adds support for "departments" property that mod-users v17.2.0 has added: https://github.com/folio-org/mod-users/commit/ffe061103c02b123cc9a6e219da30104cb690429
It skips the "preferredFirstName" property that mod-users v17.1.0 has added: https://github.com/folio-org/mod-users/commit/16b4e9845d3a349261ddd98c31d92631da794cf8
Can you add "preferredFirstName" as part of this pull request (and reword the Jira), or can you create a new Jira for that?

Comment (wwelling) [reply ramls/schemas/userdataimport.json:44]: Done

**Review by julianladisch — APPROVED** [review]

**Pull request merged**